src/TrueLayer/Payments/Model/Verification.cs

```diff
@@ -0,0 +1,22 @@
+using TrueLayer.Serialization;
+
+namespace TrueLayer.Payments.Model;
+
+public static class RemitterVerification
```

Comment:
Do you think it could make sense to rename this class to `Verification` or `PaymentVerification`? The prefix `Remitter-` has already been applied to the field names. Possibly in future we could have other types of verification that may not be strictly related to the remitter?